### PR TITLE
Refactored HTML header to be more interesting.

### PR DESCRIPTION
**Pull request by robbmanes**
Clearly, this is a vital addition to any personal GitHub pages site.

### CODE REVIEW

**Comment by wcmitchell** [comment]
Seems a worthy addition to me, clear, concise, a definite improvement over the original. 👍

File: index.html [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <!DOCTYPE html>
 <html>
 <head>
+  <!--
+  "This is github!"
+  "No... THIS.  IS.  A WEBSITE"
```
Comment (kyle-walker) [inline index.html]: I can simply not accept this. Style guide clearly indicates that all comments must be in allcaps.

**Review by kyle-walker — CHANGES_REQUESTED** [review]

**Comment by robbbTL** [comment]
Sir,

Your mistakes are as great as your love for bacon.  Reconsider this as you need all the "website" you can get.

**Review by robbmanes — COMMENTED** [review]
Resolved capitalization issue.  Note that a style guide might want to be implemented to avoid this issue in the future.

**Review by kyle-walker — APPROVED** [review]

**Pull request merged**